api/src/main/java/org/pmiops/workbench/leonardo/LeonardoRetryHandler.java

```diff
@@ -36,12 +36,21 @@ protected int getResponseCode(Throwable lastException) {
     @Override
     protected void logNoRetry(Throwable t, int responseCode) {
       if (t instanceof ApiException) {
-        logger.log(
-            getLogLevel(responseCode),
-            String.format(
-                "Exception calling Leonardo API with response: %s",
-                ((ApiException) t).getResponseBody()),
-            t);
+        if (responseCode == 404) {
+          logger.log(
+              getLogLevel(responseCode),
+              String.format(
+                  "Exception calling Leonardo API with response: %s. This is likely because a runtime "
```

Comment:
Is there any way to make this more targeted, i.e. from the call site? This is a bit broad, as it will be encountered on any call - not just GetRuntime. 

Not high priority, but preferred if you can find a way

Reply:
There's probably a way, but I don't want to spend too much effort on this ticket. If I can't figure it out before scrum I'll abandon it